tests/compiler/test_basics.py

```diff
@@ -743,6 +743,15 @@ def test_implementation_import_missing_error(snippetcompiler) -> None:
     assert exception.value.location.start_char == 20
 
 
+@pytest.mark.parametrize("name", ["", "#", " # ", "#this is a comment"])
```

Comment:
1. I think tests for methods in util should not get into the compiler folder
2.  I would also test positive examples
3. I would also test the handling of lists or files that should ignore comments